config.template

```diff
@@ -97,6 +97,14 @@ sstSubdirectory = SST
 sssSubdirectory = SSS
 mldSubdirectory = MLD
 
+# first and last year of SST climatology
+# values for preindustrial
+sstClimatologyStartYear = 1870
+sstClimatologyEndYear = 1900
+# alternative values for present day
+#sstClimatologyStartYear = 1990
+#sstClimatologyEndYear = 2011
```

Comment:
@milenaveneziani, hmm, I could understand this for comparing to another model run but for observations, there are *real* years over which those observations were taken.  I think these numbers should correspond to those real years, not to the years of the model run (which might, indeed, start with 0001).